relayer/path.go

```diff
@@ -12,8 +13,9 @@ import (
 )
 
 const (
-	check = "✔"
-	xIcon = "✘"
+	check   = "✔"
+	xIcon   = "✘"
+	EXPIRED = "EXPIRED"
```

Comment:
nit: Go stdlib and docs use camelcase for constants as well and only use a capital first letter if the constant should be exported.